src/renderer/utils/markdown.ts

```diff
@@ -157,11 +211,11 @@ const Markdown = {
 
     if ( Markdown.converter ) return Markdown.converter;
 
-    const {encodeSpecialLinks, attachment, note, tag, katex, mermaid} = Markdown.extensions;
+    const {encodeSpecialLinks, attachment, note, tag, wikilink, katex, mermaid} = Markdown.extensions;
 
     const converter = new showdown.Converter ({
       metadata: true,
-      extensions: [showdownHighlight, showdownTargetBlack, encodeSpecialLinks (),attachment (), note (), tag (), katex (), mermaid ()]
+      extensions: [showdownTargetBlack, encodeSpecialLinks (),attachment (), note (), tag (), wikilink (), showdownHighlight, katex (), mermaid ()]
```

Comment:
can you rebase? the order of plugins changed in master and this branch conflicts with it.

Reply:
fixed